bindata/v3.11.0/etcd/pod.yaml

```diff
@@ -21,10 +21,22 @@ spec:
         memory: 200Mi
         cpu: 100m
     ports:
-      - containerPort: 10257
+      - containerPort: 2379
+        name: client
+        protocol: TCP
+    ports:
+      - containerPort: 2380
+        name: peer
+        protocol: TCP
     volumeMounts:
-    - mountPath: /etc/kubernetes/static-pod-resources
-      name: resource-dir
+    - name: discovery
+      mountPath: /run/etcd/
+    - name: certs
+      mountPath: /etc/ssl/etcd/
+    - name: data-dir
+      mountPath: /var/lib/etcd/
```

Comment:
I recommend against this.  You should be managing it yourself in that staticpod resource dir

Reply:
ack